appveyor.yml

```diff
@@ -0,0 +1,6 @@
+clone_depth: 3
```

Comment:
I wont pretend to know what's all this because I dont. But isnt the number of CIs and whatever going over our heads? Since it has come to the point of having two CI configuration files at once, cant we just use one CI, you know, like sane people?

Reply:
Ok